src/cli/discord_cli.py

```
@@ -0,0 +1,278 @@
+"""
+CLI commands for Discord bot management using Typer.
+"""
+import typer
+import time
+from typing import Optional
+from rich.console import Console
+from rich.table import Table
+from rich.panel import Panel
+from rich.text import Text
+from datetime import datetime
```

Ruff (0.12.2)

`datetime.datetime` imported but unused

Remove unused import: `datetime.datetime`

(F401)